section1/exercises/strings.rb

```diff
@@ -7,9 +7,20 @@
 p "Alan Turing"
 
 # Write code that prints `Welcome to Turing!` to the terminal:
-p #YOUR CODE HERE
+<<<<<<< HEAD
```

Comment:
This line comes from a merge.. make sure you remove these otherwise your code does not run